packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -37,9 +37,20 @@ const AccountSwitcher = props => {
     const [is_deriv_demo_visible, setDerivDemoVisible] = React.useState(true);
     const [is_deriv_real_visible, setDerivRealVisible] = React.useState(true);
     const [is_dmt5_demo_visible, setDmt5DemoVisible] = React.useState(true);
-    const [is_dmt5_real_visible, setDmt5RealVisible] = React.useState(true);
+    const [is_dmt5_real_visible, setDmt5RealVisible] = React.useState(false);
 
     const wrapper_ref = React.useRef();
+    const dmt5_ref = React.useRef(null);
+
+    React.useEffect(() => {
+        if (dmt5_ref.current && is_dmt5_real_visible) {
+            dmt5_ref.current.scrollIntoView({
+                behavior: 'smooth',
+                block: 'end',
+                inline: 'nearest',
+            });
+        }
+    }, [is_dmt5_real_visible, dmt5_ref]);
```

Comment:
for me this approach didnt work error -> caught TypeError: Cannot read property ‘scrollIntoView’ of null. I may be wrong but in this case the ref doesn't need to change. I think in reality the only dependency that I need it is_dmt5_real_visible, because i need to have a side effect that runs once that variable is true. Opinion?

Reply:
That makes sense, in that case you can apply the suggestion:

> You can add an eslint exception for this if it starts yelling at you.

```suggestion
    // eslint-disable-next-line react-hooks/exhaustive-deps
    }, [is_dmt5_real_visible]);
```

Reply:
done please approve so we go ahead